package.json

```
@@ -9,19 +9,23 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@emotion/react": "^11.11.3",
+    "@emotion/styled": "^11.11.0",
+    "@mui/material": "^5.15.4",
+    "material-icons": "^1.13.12",
+    "next": "14.0.4",
     "react": "^18",
```

Mesma coisa, deixe fixa

```suggestion
    "react": "18",
```